src/client/editor/SceneLoader.js

```diff
@@ -228,6 +228,11 @@ export async function loadSerializedScene(sceneDef, baseURL, addComponent, isRoo
     throw new Error("Invalid Scene: Scene does not inherit from another scene or have a root entity.");
   }
 
+  // init scene conflict status
+  scene.conflicts = {
+    missing: false,
+    duplicate: false
+  };
```

Comment:
`conflicts` should be under `scene.userData` and should start with an underscore.